es6/2015-03/mar-24.md

```diff
@@ -625,7 +625,7 @@ DH: Suggests stage 1, but audit the space which may lead to changes.
 
 Proposal: https://github.com/leebyron/ecmascript-reverse-iterator
 
-LB: [presents slides] (found in github repo)
+LB: [presents slides] \(found in github repo\)
```

Comment:
Actually, the url for the presentation is:

https://github.com/leebyron/ecmascript-reverse-iterable/blob/master/TC39-March2015.pdf